### ISSUE TITLE
[Linux Appimage issue] No color emojis being rendered

### ISSUE THREAD

**Issue by Ranger-NF** [Demizo/Daily_You#75]
On linux, color emojis are not being rendered

![image-of-mood-icons](https://github.com/Demizo/Daily_You/assets/78952725/48cefe89-a5b8-4607-b43e-a1fbdd6ed287)

NB: I think it is an issue linked to flutter itself: [Issue on flutter repo](https://github.com/flutter/flutter/issues/119536)

Is there a workaround? Like a custom color emoji font or something?

**Comment by Demizo** [comment]
@Ranger-NF Yeah this is annoying. I will look in to if I can fix it on the Flutter side, maybe I can cheat it with a custom font on the desktop version.

**Comment by Ranger-NF** [comment]
> maybe I can cheat it with a custom font on the desktop version.
I think that's the way to go. It will cause problems with accessibility, so it should be made togglable (Personally, Colored emojis >> Non-colored)

**Comment by Demizo** [comment]
@Ranger-NF I haven't looked into this issue much yet since I am still overhauling the back-end of the app to use SAF. However, when I do get to fixing this I am curious about what Linux distro you are on? I used to have all the emojis be monochrome but since the last app update when I bumped the Flutter version I now only have the "very happy" emoji in monochrome. I am using Fedora 39, tested on both the KDE and regular GNOME spin.

**Comment by Ranger-NF** [comment]
>  I am curious about what Linux distro you are on?

Im on `Debian 12 (Bookworm) x86_64` with `Gnome 43.9` as the desktop environment.

**Comment by Ranger-NF** [comment]
I tried adding a custom color emoji (Noto color emoji to be specific). Implementing it for mood icons was easy (As it is setup as a seperate widget)

But I encountered 2 problems. One being that changing the `fontFamily` of an entire entry creates strange spacing between each word. Secondly, bundling a custom font might increase the final build size (NB: I haven't tested it yet, just a speculation). The ttf file is about 24MB on its own.

If the build size isn't a major concern, I would suggest implementing the color emoji just for mood icons just to start off (It does make a huge difference when using the app on linux)

**Comment by Ranger-NF** [comment]
I know this project is not being worked on actively, but I love this app and wanted to know if it would be possible to release it to flatpak...

If so, 
this [forum post](https://discourse.flathub.org/t/flutter-application/1096) might help...

**Comment by Demizo** [comment]
For transparency, I am still planning to package the app as a Flatpak. I have been and am still prioritizing the Android version for the near future.
